src/providers/ApolloClientProvider.tsx

```diff
@@ -27,10 +29,11 @@ export default function ApolloClentProvider({
       }),
       cache: new InMemoryCache()
     })
+
     setClient(newClient)
-  }, [config])
+  }, [config, client])
```

Comment:
I think this creates a loop. By putting `client` in the dependency array of the effect, it tells it to run whenever `client` changes. But it then sets `client` which will trigger it doing the same thing again and again